lib/wovnrb/services/html_converter.rb

```diff
@@ -4,6 +4,7 @@ def initialize(dom, store, headers)
       @dom = dom
       @headers = headers
       @store = store
+      @widget_urls = [@store.settings['api_url'] << '/widget', 'j.wovn.io', 'j.dev-wovn.io:3000']
```

Comment:
shouldn't use `<<` here. That is going to modify the `api_url` setting. Just use +. The `widget_urls` method was also OK by the way :)

Reply:
Thanks, I'll fix it.

Removing the '/v0/' from the api-url is giving me an error I'm not sure what to do with, do you have any idea?
`wovnrb/vendor/bundle/gems/minitest-5.14.1/lib/minitest/pride_plugin.rb:75:in `sub': invalid byte sequence in UTF-8`

Reply:
it's a weird error but I think it might be fixed by changing `api_translator` to use the correct url

Reply:
There were 3 places that were erroring, and editing api_translator fixed 2 of them. I'm still hunting for the last one.